api/v1alpha1/volumereplicationgroup_types.go

```diff
@@ -75,6 +81,17 @@ type VolSyncReplicationDestinationSpec struct {
 	ProtectedPVC ProtectedPVC `json:"protectedPVC,omitempty"`
 }
 
+type PeerClass struct {
+	//+optional
+	ReplicationID string `json:"replicationID,omitempty"`
+
+	//+optional
+	StorageID []string `json:"storageID,omitempty"`
+
+	//+optional
+	StorageClassName string `json:"storageClassName,omitempty"`
+}
```

Comment:
Should we move PeerClasses to drpolicy_types instead of holding it in this file?

Reply:
yes, I have moved it under DRPolicy. However VRG fields are being used in many places and it's touching more files. Thinking to taking this change in a separate PR.